packages/cache-utils/tests/digests.js

```diff
@@ -0,0 +1,52 @@
+const test = require('ava')
+
+const { pWriteFile, createTmpDir, removeFiles } = require('./helpers/main')
+
+const cacheUtils = require('..')
+
+test('Option digests', async t => {
```

Comment:
It would be nice to incorporate the expected result in the test name, e.g. `should ignore file changes when digest is provided` or something similar

Reply:
:+1: 

I just updated the test titles to make them more human-friendly.